webapp/src/DOMJudgeBundle/Resources/views/partials/scoreboard.html.twig

```diff
@@ -103,7 +103,7 @@
                             <select class="form-control mr-2" multiple size="8" name="countries[]">
                                 {% for country in filterValues.countries %}
                                     <option {% if country in scoreFilter.countries %}selected{% endif %}
-                                            value="{{ country }}">{{ country }}</option>
+                                            value="{{ country }}">{{ alpha3_countries[country] }}</option>
```

Comment:
This does get significantly longer now, if you have countries like "United Kingdom of Great Brittain and Northern Ireland". But that may be acceptable.

Reply:
Could also just shorten it in our copy of the list.

Reply:
Shortening the name would be best then, there are only a handful which are quite long though.

- Bolivia (Plurinational State of)
- British Indian Ocean Territory
- Congo (Democratic Republic of the)
- Heard Island and McDonald Islands
- Korea (Democratic People's Republic of)
- Lao People's Democratic Republic
- Micronesia (Federated States of)
- Saint Helena, Ascension and Tristan da Cunha
- Saint Vincent and the Grenadines
- South Georgia and the South Sandwich Islands
- United Kingdom of Great Britain and Northern Ireland
- United States Minor Outlying Islands
- Venezuela (Bolivarian Republic of)

Everything longer than 30 charcters got included in the above list